dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/PLL/PLL_INIT.mo

```diff
@@ -0,0 +1,29 @@
+within Dynawo.Electrical.Controls.PLL;
+
+/*
+* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model PLL_INIT "Initial model of phase locked loop"
+
+  parameter Types.VoltageModulePu U0Pu  "Start value of voltage amplitude at PLL terminal in p.u (base UNom)";
+  parameter Types.Angle UPhase0  "Start value of voltage angle at PLL terminal in rad";
+  parameter Types.PerUnit Omega0Pu "Start value of angular speed in p.u.";
```

Comment:
do you need this Omega0Pu in the init model?

Reply:
please see next comment